src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -28,14 +35,31 @@ public SwerveSubsystem(
     if (Utils.isSimulation()) {
       startSimThread();
     }
+
+    Translation2d[] kinematicsTranslations = new Translation2d[4];
+    for (int i = 0; i < 4; i++) {
+      kinematicsTranslations[i] = new Translation2d(modules[i].LocationX, modules[i].LocationY);
+    }
+    kinematics = new SwerveDriveKinematics(kinematicsTranslations);
+
+    AutoBuilder.configureHolonomic(
+        () -> super.getState().Pose,
+        (a) -> super.seedFieldRelative(a),
+        () -> getChassisSpeeds(),
+        (a) ->
+            this.setControl(
+                new SwerveRequest.FieldCentric()
+                    .withVelocityX(a.vxMetersPerSecond)
+                    .withVelocityY(a.vyMetersPerSecond)
+                    .withRotationalRate(a.omegaRadiansPerSecond)),
+        new HolonomicPathFollowerConfig(2, 0.47383085589748, new ReplanningConfig(true, true)),
```

Comment:
Lets set constants for these + program math into the radius constant (just in case we change our drivebase dimensions)